Track/Track.py

```diff
@@ -66,6 +66,21 @@ def __init__(self, parent):
     self.parent.acknowledgementText = """
 This extension was developed by the Laboratory for Translational Medicine.
 """
+    if not slicer.app.commandOptions().noMainWindow:
+      slicer.app.connect("startupCompleted()", self.installPacakges)
+    
+  def installPacakges(self):
```

Comment:
Nit: rename function "installPacakges" to "installPackages"

Reply:
@teomesrkhani I dont' have my Windows laptop with me, but it looks like it used incorrect slash (backward slash \ ) for the folder
/lib/Slicer-5.6/qt-scripted-modules**\**Data**\**2D Cine Images
Can you help play around with it if you have sometimes? I can't resolve it because I only have Mac with me for the trip. Thank you very much.

Reply:
@haphantran I reinstalled 3D Slicer for the second time and now the "Reload and Test" works without any error. Thank you.